.goreleaser-nightly.yaml

```diff
@@ -1,59 +0,0 @@
-before:
-  hooks:
-    - go mod tidy
-
-# Build a universal macOS binary
-universal_binaries:
-  - replace: false
-
-# Build the different combination of goos/arch binaries
-builds:
-  - env:
-      - CGO_ENABLED=0
-    goos:
-      - linux
-      - darwin
-    ldflags:
-      - -s -w -X 'github.com/defenseunicorns/uds-cli/src/config.CLIVersion={{.Tag}}' -X 'github.com/defenseunicorns/zarf/src/config.ActionsCommandZarfPrefix=zarf'
-      - -X 'github.com/defenseunicorns/zarf/src/config.ActionsCommandZarfPrefix=zarf'
-      - -X 'github.com/derailed/k9s/cmd.version=${K9S_VERSION}'
-      - -X 'github.com/google/go-containerregistry/cmd/crane/cmd.Version=${CRANE_VERSION}'
-      - -X 'github.com/defenseunicorns/zarf/src/cmd/tools.syftVersion=${SYFT_VERSION}'
-      - -X 'github.com/defenseunicorns/zarf/src/cmd/tools.archiverVersion=${ARCHIVER_VERSION}'
-      - -X 'github.com/defenseunicorns/zarf/src/cmd/tools.helmVersion=${HELM_VERSION}'
-    goarch:
-      - amd64
-      - arm64
-    binary: uds
-
-# Save the built artifacts as binaries (instead of wrapping them in a tarball)
-archives:
-  - format: binary
-    name_template: "{{ .ProjectName }}_{{ .Tag }}_{{- title .Os }}_{{ .Arch }}"
-
-# generate a sha256 checksum of all release artifacts
-checksum:
-  name_template: "checksums.txt"
-  algorithm: sha256
```

Comment:
Not sure if losing the checksums from go-releaser is a big deal or not for the nightly releases?